README.md

```diff
@@ -277,11 +276,25 @@ trace.getActiveSpan()?.addEvent('<error-message>', {'lumigo.type': '<error-type>
 | grpc-js | [@grpc](https://www.npmjs.com/package/@grpc) | 1.8.0~1.8.20 |
 | amqplib | [amqplib](https://www.npmjs.com/package/amqplib) | 0.9.0~0.10.3 |
 | express | [express](https://www.npmjs.com/package/express) | 4.9.0~4.18.2 |
+| ioredis | [ioredis](https://www.npmjs.com/package/ioredis) | 4.0.0~4.24.0 |
 | kafkajs | [kafkajs](https://www.npmjs.com/package/kafkajs) | 2.0.0~2.2.4 |
 | mongodb | [mongodb](https://www.npmjs.com/package/mongodb) | 3.6.6~3.7.3 |
 | | | 4.0.0~4.16.0 |
+| prisma | [prisma](https://www.npmjs.com/package/prisma) | 4.2.0~4.16.2 |
+| | | 5.0.0~5.2.0 |
```

Comment:
Why have two ranges? are there versions higher than `4.16.0` and lower than `5.0.0` that are not supported?

Reply:
we're not interested in anything that's not official (ie. -dev / -integration), take a look at [the package versions ](https://www.npmjs.com/package/prisma?activeTab=versions)

we use separate lines for major version support / when there are breaks in support (ie. if there was a 4.17.0 and a 4.18.0 and we only supported 4.18.0, then it would have to be on a new line

Reply:
OK I understand